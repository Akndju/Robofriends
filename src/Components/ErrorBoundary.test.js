import { shallow } from 'enzyme';
import React from 'react';
import ErrorBoundary from './ErrorBoundary';

it('expect to render ErrorBoundary Component', () => {
  expect(shallow(<ErrorBoundary />)).toMatchSnapshot();
});
