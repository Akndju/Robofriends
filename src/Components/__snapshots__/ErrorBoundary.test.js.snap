// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render ErrorBoundary Component 1`] = `""`;
